packages/react/src/components/Storage/FileUploader/types.ts

```diff
@@ -7,11 +7,9 @@ type LevelInfo = 'public' | 'protected' | 'private';
 export type Files = File[];
 
 export interface UploadButtonProps {
```

Comment:
We could just extend from ButtonProps here

Reply:
👍